furusiya/main.py

```diff
@@ -37,16 +39,16 @@ def main_menu():
         if choice == 1:  # load last game
             try:
                 load_game()
-            except:
+            except Exception as e:
                 msgbox('\n No saved game to load.\n', 24)
                 continue
             play_game()
         elif choice == 2:  # quit
             break
 
-if config.has("seed"):
-    seed = config.get("seed")
-    random.seed(seed)
-    print("Seeding as universe #{}".format(seed))
-    
+
+seed = config.get("seed") or int(datetime.now().timestamp())

```

Comment:
This is not necessary. I verified it on the REPL.

1) `random.seed` does not need to be called; if it isn't, it's seeded with a random seed.
2) If it's called with `None`, it gets seeded with a random seed

I would suggest removing this change and instead adding a comment: if there is no seed, this gets a random seed.

Reply:
Yes, it does get a random seed, but it doesn't print it. This does.

Reply:
Yes, but your seed may not be as good as the built-in one. Unfortunately, it seems like there's just no way to get the built-in seed.

How about `random.seed(random.random())` instead?